vyked/bus.py

```diff
@@ -97,6 +104,10 @@ def register(self):
             self._registry_client.register(self.http_host.host, self.http_host.port, self.http_host.name,
                                            self.http_host.version, self.http_host.node_id, self.http_host.clients,
                                            'http')
+        if self.ws_host:
```

Comment:
We can keep an list of services attached to the host and parse it instead of repeating this code thrice everywhere